openedx/core/djangoapps/xblock/api.py

```diff
@@ -208,13 +209,28 @@ def get_component_from_usage_key(usage_key: UsageKeyV2) -> Component:
     )
 
 
-def get_block_draft_olx(usage_key: UsageKeyV2) -> str:
+def get_block_olx(
+    usage_key: UsageKeyV2,
+    user: UserType,
+    *,
+    version: int | LatestVersion = LatestVersion.AUTO
+) -> str:
```

Comment:
Since this is technically a public API, and not marked as unstable, you should create a backwards-compatibility wrapper:


```python
def get_block_draft_olx(usage_key: UsageKeyV2) -> str:
    """ DEPRECATED. Use get_block_olx(). Can be removed post-Teak. """
    return get_block_olx(usage_key, version=LatestVersion.DRAFT)
```

Reply:
Updated https://github.com/openedx/edx-platform/pull/35932/commits/a66890d1cdc0c87464fbc15d5dbf8ebbd0ea38b2